diff -Naur gtkhotkey-0.2.1.orig/Makefile.am gtkhotkey-0.2.1/Makefile.am
--- gtkhotkey-0.2.1.orig/Makefile.am	2009-09-03 22:52:49.000000000 +0200
+++ gtkhotkey-0.2.1/Makefile.am	2011-03-11 23:37:49.670000041 +0100
@@ -11,7 +11,7 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtkhotkey-1.0.pc
 
-gtkhotkeydocdir = ${prefix}/doc/gtkhotkey
+gtkhotkeydocdir = ${datadir}/doc/gtkhotkey
 gtkhotkeydoc_DATA = \
 	README			\
 	COPYING			\
diff -Naur gtkhotkey-0.2.1.orig/Makefile.in gtkhotkey-0.2.1/Makefile.in
--- gtkhotkey-0.2.1.orig/Makefile.in	2009-09-03 22:53:33.000000000 +0200
+++ gtkhotkey-0.2.1/Makefile.in	2011-03-11 23:38:02.773333373 +0100
@@ -246,7 +246,7 @@
 SUBDIRS = src po test doc
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtkhotkey-1.0.pc
-gtkhotkeydocdir = ${prefix}/doc/gtkhotkey
+gtkhotkeydocdir = ${datadir}/doc/gtkhotkey
 gtkhotkeydoc_DATA = \
 	README			\
 	COPYING			\